graphite_beacon/handlers/pagerduty.py

```diff
@@ -30,27 +31,47 @@ def notify(self, level, alert, value, target=None, ntype=None, rule=None):
         LOGGER.debug("Handler (%s) %s", self.name, level)
         message = self.get_short(level, alert, value, target=target, ntype=ntype, rule=rule)
         LOGGER.debug('message1:%s', message)
-        if level == 'normal':
-            event_type = 'resolve'
+
+        # Extract unique alert identifiers
+        alert_name = message[message.find("<")+1:message.find(">")]
+        alert_metric = message[message.find("(")+1:message.find(")")]
+
+        # Generate hash 
+        h = hashlib.md5()
+        h.update(alert_name)
+        h.update(alert_metric)
+
+        # Use hash as incident key to support resolution
+        incident_key = h.hexdigest()
```

Comment:
Is there any benefit to `md5`ing these? Why not just do `"{alert_name:alert_metric}"`?